tools/fast/fastesting/basic_test.go

```diff
@@ -0,0 +1,38 @@
+package fastesting_test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/ipfs/go-log"
+	"github.com/stretchr/testify/require"
+
+	tf "github.com/filecoin-project/go-filecoin/testhelpers/testflags"
+	"github.com/filecoin-project/go-filecoin/tools/fast"
+	"github.com/filecoin-project/go-filecoin/tools/fast/fastesting"
+)
+
+func TestSetFilecoinOpts(t *testing.T) {
+	tf.IntegrationTest(t)
+	log.SetDebugLogging()
+
+	fastOpts := fast.FilecoinOpts{
+		DaemonOpts: []fast.ProcessDaemonOption{fast.POBlockTime(10 * time.Millisecond)},
+	}
+
+	ctx, env := fastesting.NewTestEnvironment(context.Background(), t, fastOpts)
```

Comment:
We need a call like the below to avoid leaking processes since this method starts one:
```go
	// Teardown after test ends
	defer func() {
		err := env.Teardown(ctx)
		require.NoError(t, err)
	}()
```

Reply:
good catch